pkgs/applications/system/booster/default.nix

```diff
@@ -0,0 +1,53 @@
+{ bash
+, binutils
+, buildGoModule
+, fetchFromGitHub
+, kbd
+, lib
+, libfido2
+, lvm2
+, lz4
+, makeWrapper
+, mdadm
+, unixtools
+, xz
+, zfs
+}:
+
+buildGoModule rec {
+  pname = "booster";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "anatol";
+    repo = pname;
+    rev = version;
+    hash = "sha256-kalVFVBb+ngoUpm+iiIHGS6vBVLEvTVyKuSMSMbp7Qc=";
+  };
+
+  vendorHash = "sha256-GD+nsT4/Y2mTF+ztOC3N560BY5+QSfsPrXZ+dJYtzAw=";
+
+  postPatch = ''
+    substituteInPlace init/main.go --replace "/usr/bin/fsck" "${unixtools.fsck}/bin/fsck"
+  '';
+
+  # integration tests are run against the current kernel
+  doCheck = false;
+
+  nativeBuildInputs = [ kbd makeWrapper lz4 xz ];
```

Comment:
I am pretty sure we want to move kbd, lz4 and xz to buildInputs.

Reply:
should I add them to both? the program shells out during testing, hence my inclusion of them here